fdbserver/DataDistribution.actor.h

```diff
@@ -75,12 +75,16 @@ struct GetTeamRequest {
 	bool wantsNewServers;
 	bool wantsTrueBest;
 	bool preferLowerUtilization;
+	bool requiresAssignedData;
```

Comment:
I would appreciate if a comment can be added to explain `requiresAssignedData `.
Although I can figure this out by reading this PR, the future me or someone else will take more time to understand this.

Reply:
I'd rather this be clear from the name of the variable than add a comment. I could change it to `teamMustHaveShards` if that is more obvious, or I'm open to other suggestions.

Reply:
`teamMustHaveShards ` is better and good to me.